### ISSUE TITLE
Liste des différentes features à développer

### ISSUE THREAD

**Issue by LucasWernerCESI** [LucasWernerCESI/station_meteo#5]
- Module d'acquisition des données (code Python sur l'ESP pour envoi des relevés vers l'API)
- API CRUD architecture REST (Saisie, modification, lecture et suppression de données en BDD)
- Connecteur BDD (méthodes ou fonctions de gestion de la connexion et des requêtes BDD)
- Application Web pour consulter les relevés (graphiques dynamiques, plan de la maison dynamique ?)
- Conception BDD et requêtes SQL
- BONUS : Automatisation de l'installation d'une nouvelle sonde
- [x] Prendre une décision

**Comment by LucasWernerCESI** [comment]
- Module d'acquisition des données (code Python sur l'ESP pour envoi des relevés vers l'API)
- API CRUD architecture REST (Saisie, modification, lecture et suppression de données en BDD)
- Connecteur BDD (méthodes ou fonctions de gestion de la connexion et des requêtes BDD)
- Application Web pour consulter les relevés (graphiques dynamiques, plan de la maison dynamique ?)
- Conception BDD et requêtes SQL
- Automatisation de l'installation d'une nouvelle sonde ?

**Comment by NicoCesi789** [comment]
L'idée d'automatisation d'une nouvelle sonde me plaît bien... C'est à voir dans un second temps peut être.

**Comment by LucasWernerCESI** [comment]
Ouais en bonus. J'ajouterai à la liste la conception des schémas BDD et l'écriture des requêtes SQL. Même si tu l'as déjà fait au moins ça apparaitra dans le suivi.

**Comment by YannProject** [comment]
*Application Web* tu parles d'une page web en Html  ( Bootstrap ) etc ?

**Comment by LucasWernerCESI** [comment]
Une page Web HTML/CSS mise à jour en AJAX, avec Bootstrap pour faciliter le responsive oui

**Comment by YannProject** [comment]
Ok parfait donc ce serait la page web qui sera afficher sur l'ecran du raspberry a la fin du projet ?

**Comment by LucasWernerCESI** [comment]
Oui. Mise à jour via AJAX pour éviter une partie de code serveur python supplémentaire ^^
